gaphor/ui/namespace.py

```diff
@@ -504,7 +504,7 @@ def search_func(model, column, key, iter, data=None):
             Gtk.DestDefaults.ALL, [NamespaceView.DND_TARGETS[-1]], Gdk.DragAction.MOVE
         )
         self.connect("drag-motion", NamespaceView.on_drag_motion)
-        self.connect("drag-drop", NamespaceView.on_drag_drop)
+        # self.connect("drag-drop", NamespaceView.on_drag_drop)
```

Comment:
Can we remove this line since you deleted the method?

Reply:
> These changes look really good. The drag and drop is now working great in the Namespace. I did notice now when dragging from the toolbox that it is dropping two of each item on to the canvas.

I can not reproduce this behaviour on my side (GTK+3 on MacOS).